sycl/plugins/cuda/CMakeLists.txt

```diff
@@ -1,29 +1,11 @@
 message(STATUS "Including the PI API CUDA backend.")
 
- # cannot rely on cmake support for CUDA; it assumes runtime API is being used.
- # we only require the CUDA driver API to be used
- # CUDA_CUDA_LIBRARY variable defines the path to libcuda.so, the CUDA Driver API library.
-
-find_package(CUDA 10.1 REQUIRED)
-
-# Make imported library global to use it within the project.
-add_library(cudadrv SHARED IMPORTED GLOBAL)
-
-if (WIN32)
```

Comment:
I'm not sure of the significance of this part, but should/have you checked whether the removal of this code affects windows builds?

Reply:
Do we officially support windows builds in CUDA? CI doesn't seem to run it.

This code was moved to unified-runtime repo: https://github.com/oneapi-src/unified-runtime/blob/adapters/source/adapters/cuda/CMakeLists.txt

It was causing issues because cudadrv was being imported twice. I think the behaviour should stay the same after the move. 

For what is worth, the windows job passed with this changes: https://github.com/intel/llvm/actions/runs/6340282334

Reply:
Windows support is experimental for cuda backend but I don't think it is tested by any CI. You should be able to build and run most tests on it though. Would be good to just check if still builds if the CMake changes could affect it. Codeplay has a windows machine available for this. If you are sure it won't affect windows build then of course this won't be worthwhile.